### PR TITLE
Fix Orders Request

### PR DESCRIPTION
**Pull request by Osama-you**

### CODE REVIEW

File: server/database/queries/user/getUserOrders.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ const connection = require('../../config/connection');
 const getUserOrders = ({ userId }) => {
   const sql = {
     text:
-      "SELECT R.description, R.date, U.username, U.mobile, U.avatar, U.location, P.title, P.bio, P.price_hour, P.price_hour, P.rating, P.cover_image, O.id, O.state, O.arrive_time FROM orders_request AS R INNER JOIN users AS U ON R.provider_id = U.id INNER JOIN providers AS P ON R.provider_id = P.user_id INNER JOIN orders AS O ON R.id = O.orders_request_id WHERE R.user_id = $1 AND R.state='accepted';",
+      "SELECT R.description, R.date, U.username, U.mobile, U.avatar, U.location, P.title, P.bio, P.price_hour, P.price_hour, P.rating, P.cover_image, P.service_type, O.id, O.state, O.arrive_time FROM orders_request AS R INNER JOIN orders AS O ON R.id = O.orders_request_id INNER JOIN users AS U ON R.provider_id = U.id INNER JOIN providers AS P ON R.provider_id = P.user_id WHERE R.user_id = $1 AND R.state='accepted';",
```
Comment (mAbadsa) [inline server/database/queries/user/getUserOrders.js]: P.price_houre is duplicated

File: server/database/queries/user/getUserOrdersReq.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ const connection = require('../../config/connection');
 const getUserOrdersReq = ({ userId }) => {
   const sql = {
     text:
-      "SELECT R.id, R.description, R.date, U.username, U.mobile, U.avatar, U.location, P.title, P.bio, P.price_hour, P.price_hour, P.rating, P.cover_image FROM orders_request AS R INNER JOIN users AS U ON R.provider_id = U.id INNER JOIN providers AS P ON  R.provider_id = P.id WHERE R.user_id = $1 AND R.state='pending';",
+      "SELECT R.id, R.description, R.date, U.username, U.mobile, U.avatar, U.location, P.title, P.bio, P.price_hour, P.price_hour, P.rating, P.cover_image, P.service_type FROM orders_request AS R INNER JOIN users AS U ON R.provider_id = U.id  INNER JOIN providers AS P ON R.provider_id = P.user_id WHERE R.user_id = $1 AND R.state='pending';",
```
Comment (mAbadsa) [inline server/database/queries/user/getUserOrdersReq.js]: P.price_houre is duplicated

File: server/database/queries/user/getUserOrders.js (hunk 1 above)
Comment (Osama-you) [reply server/database/queries/user/getUserOrders.js]: done

File: server/database/queries/user/getUserOrdersReq.js (hunk 2 above)
Comment (Osama-you) [reply server/database/queries/user/getUserOrdersReq.js]: done

**Pull request merged**